src/lcpubkey.cpp

```diff
@@ -25,6 +25,9 @@ namespace {
     typedef CryptoPP::RSASS<CryptoPP::PSSR, LcRSAHashFunction>::Signer RSA_Recovery_Signer;
     typedef CryptoPP::RSASS<CryptoPP::PSSR, LcRSAHashFunction>::Verifier RSA_Recovery_Verifier;
 
+    typedef CryptoPP::RSASS<CryptoPP::PSSR, CryptoPP::SHA1>::Verifier RSA_PSSR_SHA1_Verifier;
+    typedef CryptoPP::RSASS<CryptoPP::PSSR, CryptoPP::SHA1>::Signer RSA_PSSR_SHA1_Signer;
```

Comment:
Is there a specific reason for adding the `RSA_PSSR_SHA1` verifier and signer
instead of changing the `LcRSAHashFunction` to `CryptoPP::SHA1` within your project?
(`RSA_Recovery_Verifier` is `CryptoPP::RSASS<CryptoPP::PSSR, CryptoPP::SHA256>`)

I know the interface is really limited. Back then my decision was to hide everything away in the implementation, today I would probably do that differently and have a templated verify method.